docs/src/main/paradox/healthcheck.md

```diff
@@ -0,0 +1,24 @@
+# Health check
+
+A [health check for Akka Management](https://doc.akka.io/docs/akka-management/current/healthchecks.html)
+is provided. To enable it you need to add the following configuration
```

Comment:
we could consider enabling it by default in reference.conf, and instead document how to disable it?

Reply:
I think a readiness check for databases is controversial enough to let users enable it if they want it